src/delaunay.js

```diff
@@ -173,7 +173,9 @@ export default class Delaunay {
     this.renderHull(context);
     return buffer && buffer.value();
   }
-  renderPoints(context, r = 2) {
+  renderPoints(context, r) {
+    if (r === undefined  && (!context || typeof context.moveTo !== "function")) r = context, context = null;
+    r = (r == null) ? 2 : +r;
```

Comment:
In the `renderPoints(context, null)` case, we still want `r` to be equal to `+null` = `0`. So how about a tiny tweak…

```suggestion
    if (r === undefined && (!context || typeof context.moveTo !== "function")) r = context, context = null;
    r = r == undefined ? 2 : +r;
```